public/css/security-tips.css

```diff
@@ -52,18 +56,30 @@
   margin-right: 1rem;
 }
 
-.article-icon.how-hackers-work {
-  background-color: var(--orange6);
+/* .how-hackers-work {
+  color: var(--orange6);
+}
+
+.strong-passwords {
+  color: var(--blue3);
+}
+
+.after-breach {
+  color: var(--violet3);
 }
 
-.article-icon.strong-passwords {
-  background-color: var(--blue3);
+.steps-to-protect {
+  color: #008787;
 }
 
-.article-icon.after-breach {
-  background-color: var(--violet3);
+.five-myths {
+  color: var(--purple3);
 }
 
+.next-steps {
+  color: #e31587;
+} */
```

Comment:
Should we just delete all this? Or are we using some CSS minifier that will remove it anyways?